lib/iris/tests/unit/cube/test_Cube__operators.py

```diff
@@ -160,5 +161,28 @@ def test_div_cubes(self):
         self.assertTrue(isinstance(data, np.ndarray))
         self.assertEqual(data.shape, ())
 
+
+class Test_Masked_Lazy_Maths(tests.IrisTest):
+
+    def build_lazy_cube(self):
+        data = ma.array([[1., 1.], [1., 100000.]], mask=[[0, 0], [0, 1]])
+        data = biggus.NumpyArrayAdapter(data)
+        cube = iris.cube.Cube(data, standard_name='air_temperature', units='K')
+        lat = iris.coords.DimCoord([-10, 10], 'latitude')
+        lon = iris.coords.DimCoord([10, 20], 'longitude')
+        cube.add_dim_coord(lat, 0)
+        cube.add_dim_coord(lon, 1)
+        return cube
+
+    def test_add(self):
+        cube_a = self.build_lazy_cube()
+        cube_b = self.build_lazy_cube()
+        cube_c = cube_a - cube_b
```

Comment:
&#161;Pedantry alert! @ajdawson I'm really sorry but this test method is called `test_add` and you're subtracting your two cubes... :sweat: